test_apps/pnpm/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "license": "UNLICENSED",
+  "private": true,
+  "scripts": {
+    "dev-server": "encore dev-server",
+    "dev": "encore dev",
+    "watch": "encore dev --watch",
+    "build": "encore production --progress"
+  },
```

Comment:
Remove `scripts` just to keep this file as simple as possible (with nothing extra)